setup.py

```diff
@@ -37,7 +37,7 @@
 REQUIREMENTS_INSTALL = [
     "openscm-units>=0.2,<0.3",
     "pint",
-    "pyam-iamc>=0.5.0",
+    "pyam-iamc>=0.5.0, <0.11",
```

Comment:
I think the ```plot()``` call in ```pyam``` was added in v0.10.

Reply:
OK, I've pinned to v0.10+